src/lib/registration/registration.rb

```diff
@@ -167,6 +167,18 @@ def migration_products(installed_products)
       migrations
     end
 
+    def offline_migration_products(installed_products, target_base_product)
+      log.info "Offline migration for: #{target_base_product}."
+      migration_paths = []
+      ConnectHelpers.catch_registration_errors do
+        migration_paths = SUSE::Connect::YaST
+                          .system_offline_migrations(installed_products, target_base_product)
+      end
```

Comment:
does it make sense to continue if there are not migration paths?

Reply:
That's handled [elsewhere](https://github.com/yast/yast-registration/pull/353/files#diff-1be991fe1b75a62325e272cdd2ee099dR301).